treecraft/core/file_handlers.py

```
@@ -0,0 +1,399 @@
+import os
+from Bio import Phylo, SeqIO
+from PyQt6.QtWidgets import QFileDialog, QMessageBox
+from PyQt6.QtSvg import QSvgGenerator
+from PyQt6.QtCore import Qt
```

Ruff (0.8.2)

`PyQt6.QtCore.Qt` imported but unused

Remove unused import: `PyQt6.QtCore.Qt`

(F401)